app/javascript/packs/usersEventList.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from "react";
+import ReactDOM from "react-dom";
+import axios from "axios";
+
+class UsersEventList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: "",
+      addedUsers: [],
+      users: []
+    };
+  }
+
+  componentWillMount() {
+    axios
+      .get("/api/v1/users.json", {}, { "Content-Type": "application/json" })
```

Comment:
If the request hits an API controller it is probably unnecessary to include `.json` at the end of the path. Do `"/api/v1/users"` instead of `"/api/v1/users.json"`